client/i18n/locales/english/translations.json

```
@@ -325,7 +325,11 @@
     "sorry-getting-there": "Sorry, your code does not pass. You're getting there.",
     "sorry-hang-in-there": "Sorry, your code does not pass. Hang in there.",
     "sorry-dont-giveup": "Sorry, your code does not pass. Don't give up.",
-    "challenges-completed": "{{completedCount}} of {{totalChallenges}} challenges completed"
+    "challenges-completed": "{{completedCount}} of {{totalChallenges}} challenges completed",
+    "season-greetings-fcc": "Season's Greetings from the freeCodeCamp community 🎉",
+    "if-getting-value": "If you're getting a lot out of freeCodeCamp, now is a great time to donate to support our nonprofit's mission.",
+    "building-a-university": "We're Building a Free Computer Science University Degree Program",
+    "if-help-univeristy": "We've already made a ton of progress. Support our charity with the long road ahead."
```

_"with"_ sounded weird.

```suggestion
    "if-help-univeristy": "We've already made a ton of progress. Support our charity for the long road ahead."
```

---

Quincy's copy. Should I double check with him?

---

That, or edit the suggestion to just fix the `university` typo